src/Core/StudentsDashboard.Application/Authentication/Commands/Login/LoginHandler.cs

```diff
@@ -0,0 +1,33 @@
+using ErrorOr;
+using MediatR;
+using StudentsDashboard.Application.Common.Errors;
+using StudentsDashboard.Application.Common.Interfaces.Authentication;
+using StudentsDashboard.Application.Contracts.Authentication;
+using StudentsDashboard.Application.Persistance;
+
+namespace StudentsDashboard.Application.Authentication.Commands.Login;
+
+public class LoginHandler : IRequestHandler<LoginCommand, ErrorOr<LoginResponse>>
```

Comment:
Klasa powinna nazywać się LoginCommandHandler, chodzi o to że handler czyli "uchwyt" nawiązuje do uchwycenia "komendy loginu" czy jak to nazwać, nie natomiast do samego loginu